internal/translayer/client.go

```diff
@@ -0,0 +1,29 @@
+package translayer
```

Comment:
suggestion: `translayer` implies two terms, "translation" and "layer". The  latter is somewhat obvious as every package is a layer. How about just `translation`? That naming is pretty idiomatic and immediately implies what is done here, namely translating things.

Reply:
coming on next PR update